src/vmm/src/device_manager/mmio.rs

```diff
@@ -35,46 +35,40 @@ use vm_memory::GuestAddress;
 /// Errors for MMIO device manager.
 #[derive(Debug)]
 pub enum Error {
```

Comment:
Could `thiserror::Error` be used on `Error` here to avoid needing to write the `Display` implementation manually?

Reply:
Done, created a separate commit.